tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -349,6 +362,25 @@ test = Matrix(test_cut[:, remove_names]);
 end;
 ```
 
+```julia
```

Comment:
Why are we defining two models here?

As far as I can tell, the previous definition of `linear_regression` is not being used?

Reply:
In the old version, the `linear_regression` function has signature `(x, y, n_obs, n_vars)`, but `n_obs` is not used because of the use of `y ~ MvNormal(mu, σ² * I)`; in the newly defined `linear_regression(x, y, n_obs, n_vars)`, it iterates through `n_obs` and uses `y[i] ~ Normal(mu[i], sqrt(σ²))` (though could be less efficient). So I kept both versions for user to choose.
It seems not necessary to keep both - I just removed the first function.